src/EFTools/EntityDesign/VisualStudio/ModelWizard/ModelObjectItemWizard.cs

```diff
@@ -298,6 +302,20 @@ private static void PopluateReplacementDictionary(Project project, Dictionary<st
         [SuppressMessage("Microsoft.Maintainability", "CA1506:AvoidExcessiveClassCoupling")]
         public void RunFinished()
         {
+#if VS14ORNEWER
+            TelemetryService.DefaultSession.PostEvent(
+                new TelemetryEvent("vs/entityframework/entitydatamodelwizard")
+                {
+                    Properties =
+                    {
+                        {
+                            "VS.EntityFramework.EntityDataModelWizard.GenerationOption",
+                            _modelBuilderSettings.GenerationOption
```

Comment:
We may want to split this out into two properties: IsFromDatabase & IsCodeFirst (or similar)